website/src/components/HeaderIcons/HeaderIconstest.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Same thing for naming these tests most likely dont get called

Reply:
Those are failed cases (require ur pr) therefore, I don't want npm test detect them